types/plugins/sentinel/config.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The HTNN Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sentinel
+
+import (
+	"mosn.io/htnn/api/pkg/filtermanager/api"
+	"mosn.io/htnn/api/pkg/plugins"
+)
+
+const (
+	Name = "sentinel"
+)
+
+func init() {
+	plugins.RegisterPluginType(Name, &Plugin{})
+}
+
+type Plugin struct {
+	plugins.PluginMethodDefaultImpl
+}
+
+func (p *Plugin) Type() plugins.PluginType {
+	return plugins.TypeTransform
```

Comment:
Better to use Traffic:
https://github.com/mosn/htnn/blob/81bc6afba12df99b4ef46766c91a30b45bd1bf65/api/pkg/plugins/type.go#L27

Reply:
My carelessness. Done